app/src/main/java/com/keylesspalace/tusky/adapter/StatusViewHolder.java

```diff
@@ -92,11 +95,88 @@ protected void setupWithStatus(StatusViewData.Concrete status, final StatusActio
             }
 
             rebloggedBar.setOnClickListener(v -> listener.onOpenReblog(getAdapterPosition()));
+        }
+        setContentDescription(status);
+        // Workaround for RecyclerView 1.0.0 / androidx.core 1.0.0
+        // RecyclerView tries to set AccessibilityDelegateCompat to null
+        // but ViewCompat code replaces is with the default one. RecyclerView never
+        // fetches another one from its delegate because it checks that it's set so we remove it
+        // and let RecyclerView ask for a new delegate.
+        itemView.setAccessibilityDelegate(null);
+    }
 
+    private void setContentDescription(@Nullable StatusViewData.Concrete status) {
+        if (status == null) {
+            itemView.setContentDescription(
+                    itemView.getContext().getString(R.string.load_more_placeholder_text));
+        } else {
+            setStatusDescriptionForStatus(status);
         }
 
     }
 
+    private void setStatusDescriptionForStatus(@NonNull StatusViewData.Concrete status) {
+        CharSequence postedAgo = DateUtils.getRelativeTimeSpanString(
```

Comment:
Shouldn't this respect the "use absolute time" setting?

Reply:
Yep, we probably should.